papers/bcr-0004-bc32.md

```diff
@@ -0,0 +1,51 @@
+# The BC32 Data Encoding Format
+## BCR-0004
+
+**© 2020 Blockchain Commons**
+
+Author: Wolf McNally<br/>
+Date: May 2, 2020
+
+---
+
+### Introduction
+
+The [Bech32] checksummed base32 format was introduced to transport segregated witness signatures for Bitcoin. It has several desirable qualities:
+
+* The payload and checksum data use a limited character set compatible with both URI syntax and QR code alphanumeric encoding mode.
+* The character set uses the letters A-Z in a case-modal fashion (i.e., one can use all upper case or all lower case but the algorithm rejects mixed-case.)
+* The character set is chosen to minimize ambiguity, and the ordering is chosen to minimize the number of pairs of similar characters.
+* Built-in error detection.
```

Comment:
…error detection, and option error identification for smaller <80 character payloads.

Reply:
Fixed.